macros/creators/bmcv.xml

```diff
@@ -0,0 +1,5 @@
+<macros>
+    <xml name="creator">
+        <organization name="Biomedical Computer Vision Group" alternateName="BMCV" url="http://www.bioquant.uni-heidelberg.de/research/groups/biomedical_computer_vision.html" />
```

Comment:
probably good to have a yield here.

Reply:
Hm I have looked into [`yield`](https://planemo.readthedocs.io/en/latest/writing_advanced.html#parameterizing-xml-macros-with-yield) but I do no see in what way you mean to use it here?